src/main/java/cz/xtf/radanalytics/util/TestHelper.java

```diff
@@ -160,4 +169,50 @@ private static Waiter areNoPodsPresent(Supplier<List<Pod>> podSupplier) {
 		return pods -> pods.stream().count() == 0L;
 	}
 	////////// =================
+
+	public static String downloadAndGetResources(String localWorkDir, String templateFileName, String resourcesUrl) {
+		log.info("Downloading and getting Resources from {}", resourcesUrl);
+		if (RESOURCES == null) {
```

Comment:
This wont work, if you are going to download multiple resources. It will keep only the first one. This functionality should be in the specific class, that is using this method, ie. Oshinko.java